### DISCUSSION TITLE
ModuleNotFoundError: No module named 'bs4'

### DISCUSSION THREAD

**Discussion by Omaralsaabi** [safevideo/autollm#148] [Q&A] [answered]
from bs4 import BeautifulSoup
ModuleNotFoundError: No module named 'bs4'

please add BeautifulSoup lib to the package.

**Comment by talhasarit** [comment, marked as answer]
Hello @Omaralsaabi,

Thank you very much for your input; we greatly appreciate your contribution to our project. For the issue with `BeautifulSoup`, we recommend installing `autollm[readers]`. This not only includes `BeautifulSoup` but also other specific packages required by various readers in our package:

```bash
pip install autollm[readers]
```

Your feedback is invaluable to us. Please feel free to reach out if you need any more help.

Best wishes,
Talha
